st-embroider/2024/2024-05-28.md

```diff
@@ -0,0 +1,61 @@
+# 2024-05-28
+
+Note Taker: Chris Manson
+Time Keeper: Alex
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+- [x] Ed Falkner (EF)
+- [x] Chris Manson (CM)
+- [ ] Aaron Chambers (AC)
+- [x] Alex (A)
+- [ ] Preston Sego (PS)
+- [x] Simon Ihmig (SI)
+- [ ] Katie Gengler (KG)
+- [ ] Peter Meehan (PM)
+- [ ] Krystan HuffMenne (KH)
+- [x] Peter Wagenet (PW)
+- [ ] James Davis (JD)
+- [x] Marine Dunstetter (MD)
+- [x] David Taylor
+- [x] tommyjr
+
+## Topics
+
+### https://github.com/embroider-build/ember-auto-import/pull/626  eai windows woes <@void-mAlex>
+
+- discovered an issue with sharedInternals for windows
+- we need to fix it on stable
+
+### discussion of an api for macros isTesting() to deal with a public API for production && force build tests <@mansona> 
+
+- we need to be able to run production build with tests
+- (discussion)
+- CM: is this something that should go into a babel config
+- EF: yea maybe
+- there could be some other changes that would need to happen
+- we might want to have an embroider config file
+- if vite and webpack both care about a single setting then it would be great that they exist in a different file
+- (looking into the code)
+- it looks like we can probaly remove the macros global `mode` now that each macro can express if it relies on runtime info
+- CM: how do we move forward?
+- EF: maybe the config in meta is the simpler version
+- i would like it to be a micro library `@embroider/config-meta` is fine
+
+
+### https://github.com/embroider-build/addon-blueprint/issues/288  testing components from engines faces the same issues Looks like re-exporting from a dummy app works within classic and embroider optimized world <@void-mAlex>
+
+- (back and forth discussion about the details of the problem)
+- CM: maybe we just don't support embroider optimized
+- A: we can support it if we have one weird trick of rexporting private things in an engine
```

Comment:
```suggestion
- A: we can support it by doing a re-export from the dummy app folder which would only apply to tests and due to importing it as a path to the v1 addon (`yourengine/components/yourcomponent`) it would continue to work in the future
```